sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,196 @@
 import pytest
 from azure.eventhub.extensions.checkpointstoretable import TableCheckpointStore
+from azure.data.tables import TableServiceClient
+import uuid
+import warnings
+import random
+import os
+from azure.eventhub.exceptions import OwnershipLostError
 
+STORAGE_CONN_STR = [os.environ.get("AZURE_STORAGE_CONN_STR", "Azure Storage Connection String")]
 
-def test_constructor():
-    client = TableCheckpointStore()
-    assert client is not None
-    
+def get_live_storage_table_client(storage_connection_str):
+    try:
+        table_name = chr(random.randint(ord('a'), ord('z'))) +(uuid.uuid4().hex)
+        table_service_client = TableServiceClient.from_connection_string(storage_connection_str)
+        table_service_client.create_table_if_not_exists(table_name)
```

Comment:
based on this method name, your table name shouldn't result in any errors anyway ;)

Reply:
I think it does cause it need to be alphanumeric